app/models/comment.rb

```diff
@@ -0,0 +1,5 @@
+class Comment < ApplicationRecord
+
```

Comment:
- [ ] Validations should be present for comment body. -- Let's add it.

Reply:
fix